api/rda.py

```diff
@@ -71,6 +72,30 @@ def wrapper(body, **kwargs):
     return wrapper
 
 
+def endpoints(plugin=None):
+    plugins_list = ["epos", "gbif", "digital_csic", "dspace7", "signposting"]
+    plugins_with_endpoint = []
+    links = []
+
+    for plug in plugins_list:
+        try:
+            config = configparser.ConfigParser()
+            config.read("plugins/" + plug + "/config.ini")
+            links.append(config["Generic"]["endpoint"])
+            plugins_with_endpoint.append(plug)
+        except:
+            print("No endpoint found for " + plug)
```

Comment:
Use logging.debug or logging.info (as you see fit)

Reply:
Used debug